### ISSUE TITLE
Import via Web UI silently fails when new records violate an index constraint

### ISSUE THREAD

**Issue by lacikawiz** [arangodb/arangodb#10470]
## My Environment

* __ArangoDB Version__:       3.5.0
* __Storage Engine__:              RocksDB 
* __Deployment Mode__:        Single Server 
* __Deployment Strategy__:   Manual Start 
* __Configuration__:               Simple basic setup with defaults
* __Infrastructure__:               Own server
* __Operating System__:        Ubuntu 18.04
* __Total RAM in your machine__:        8Gb. 
* __Disks in use__:        SSHD 
* __Used Package__:              Built from source (no changes)

## Component, Query & Data

__Affected feature__:
Import data into collection via web interface (aardvark)

__AQL query (if applicable)__:

__AQL explain (if applicable)__:

__Dataset__:
Simple records like this: 
```JSON
{
  "Name": {
    "ru": "Фильтр масляный двигателя",
    "en": "Engine oil filter"
  },
  "src": {
    "existru": {
      "lastcheck": 0,
      "id": "7"
    }
  }
}
```

__Size of your Dataset on disk__:
n/a

__Replication Factor & Number of Shards (Cluster only)__:
n/a

## Steps to reproduce

1.  Log in to the Web UI (eg: localhost::8529 ), select your database, and create an empty document collection. 
2.  Set up a hash index on a collection with the "unique" option set
3.  Import a JSON file into the collection which is either missing the values for the index, or have duplicate values

__Problem__:
The documents which violate the constraint are not getting imported. Eg, if the key of the index is missing then only one record will be imported, all others won't. 
**The UI gives no error message, or any kind of indication as to why the documents are not imported.** Not even checking the network traffic reveals any information.

__Expected result__:
Provide an error message, something to the effect "document _x_ was not imported due to unique constraint violation - in index _xyz_"

**Comment by OmarAyo** [comment]
Hi @lacikawiz 

Thanks for reporting this, we reproduce this internally and we will fwd your request internally

By the way, if you try to execute the import operation from `arangoimp` you will get a proper error message - something like:
```
2019-11-20T11:47:36Z [14825] WARNING [e5a29] at position 0: creating document failed with error 'unique constraint violated', offending document: {"_key":"356","_id":"test/356","_rev":"_ZmgcgBC--_","Nam1":"Omar"}
```
Best

**Comment by maxkernbach** [comment]
Hi @lacikawiz,

Version 3.5.3, which includes a fix for this issue, has been released.

From the [Changelog](https://github.com/arangodb/arangodb/blob/9a9c14918bc5cea2c9c121580fa925416abc1223/CHANGELOG#L29):
> * Fixed issue #10470: The WebUI now shows potential errors and details which
  occured using _api/import (e.g. unique constraint violated).

Please find the latest release in the [download section](https://www.arangodb.com/download/).
